src/algokit/core/tasks/mint/mint.py

```diff
@@ -205,11 +197,11 @@ def mint_token(  # noqa: PLR0913
         sp=client.suggested_params(),
         reserve=_reserve_address_from_cid(metadata_cid) if mutable else "",
         unit_name=unit_name,
-        asset_name=asset_name,
+        asset_name=token_metadata.name,
         url=_create_url_from_cid(metadata_cid) + "#arc3" if mutable else "ipfs://" + metadata_cid + "#arc3",
         manager=creator_account.address if mutable else "",
         total=total,
-        decimals=decimals,
+        decimals=token_metadata.decimals,
```

Comment:
Do we want to ensure a decimals value is always supplied?

Reply:
There is always a default of 0 for it.